lib/cog/repository/bundles.ex

```diff
@@ -464,7 +464,7 @@ defmodule Cog.Repository.Bundles do
     from bv in BundleVersion,
     join: b in assoc(bv, :bundle),
     where: b.name == ^site,
-    where: bv.version == @permanent_site_bundle_version,
+    where: bv.version == ^@permanent_site_bundle_version,
```

Comment:
That's interesting... I'd thought the compiler would see that the same as putting a string literal in there.

In any event, good find 👍